sky/shell/platform/ios/framework/Source/FlutterViewController.mm

```diff
@@ -435,11 +451,37 @@ - (UIStatusBarStyle)preferredStatusBarStyle {
 
 - (void)dealloc {
   [[NSNotificationCenter defaultCenter] removeObserver:self];
+  [super dealloc];
+}
 
-  [_dynamicServiceLoader release];
-  [_dartProject release];
+#pragma mark - Application Messages
 
-  [super dealloc];
+- (void)sendString:(NSString*)message
+   withMessageName:(NSString*)messageName {
+  _appMessageSender->SendString(messageName.UTF8String, message.UTF8String,
```

Comment:
Sanity check arguments since this interface is available to third parties.

* `message` may be nil.
* `messageName` may be nil.
* In the next method, `callback` may be nil.